scripts/cmake_configurator/parser/cmake_parser.py

```diff
@@ -9,7 +8,7 @@ def parse(self, file_path: str) -> list[cmake_variable]:
       variables ready to be displayed
       """
       comment: str = ""
-      self.file_lines = open(dirname(__file__) + file_path).readlines()
+      self.file_lines = open(file_path).readlines()
```

Comment:
Please squash this commit into the other one. Also, what happens with the file handle returned by open(..) ? When do we close it? Shouldn't we use with open(...) as fd etc. ?

Reply:
You are right on both of those. I'm fixing it right now.